mvlearn/plotting/plot.py

```diff
@@ -135,10 +148,116 @@ def crossviews_plot(
     else:
         return (fig, axes)
+
+
+def quick_visualize(
+    Xs,
+    labels=None,
+    figsize=(5, 5),
+    title=None,
+    cmap=None,
+    show=True,
+    context="notebook",
+    ax_ticks=True,
+    ax_labels=True,
+    scatter_kwargs={},
+    fig_kwargs={},
+):
+    r"""
+    Computes common principal components using MVMDS for dimensionality
+    reduction and plots the multi-view data on a single 2D plot for easy
+    visualization. This can be thought of as the multi-view analog of
+    using PCA to decompose data and plot on principal components.
+
+    See Also
+    --------
+    mvlearn.embed.MVMDS
+
+    Parameters
+    ----------
+    Xs : list of array-likes or numpy.ndarray
+        - Xs length: n_views
+        - Xs[i] shape: (n_samples, n_features_i)
+        The multi-view data to reduce to a single plot.
+    labels : boolean, default=None
+        Sets the labels of the samples.
+    figsize : tuple, default=(5,5)
+        Sets the figure size.
+    title : string, default=None
+        Sets the title of the figure.
+    cmap : String, default=None
+        Colormap argument for matplotlib.pyplot.scatter.
+    show : boolean, default=True
+        Shows the plots if true. Returns the objects otherwise.
+    context : one of {'paper', 'notebook', 'talk', 'poster, None},
+        default='notebook'
+        Sets the seaborn plotting context.
+    ax_ticks : boolean, default=True
+        Whether to have tick marks on the axes.
+    ax_labels : boolean, default=True
+        Whether to label the axes with the view and dimension numbers.
+    scatter_kwargs : dict, default={}
+        Additional matplotlib.pyplot.scatter arguments.
+    fig_kwargs : dict, default={}
+        Additional matplotlib.pyplot.figure arguments.
+
+    Returns
+    -------
+    fig : figure object
+        Only returned if `show=False`.
+
+    Notes
+    -----
+    This function simply uses ``MVMDS`` with ``n_components=2`` to
+    reduce arbitrarily many views of input data to 2-dimensions, then
+    makes a scatter plot.
+
+    .. figure:: /figures/quick_visualize.png
+        :width: 250px
+        :alt: Quick Visualization of Multi-view Data
+        :align: center
+
+    """
+    Xs = check_Xs(Xs)
+
+    mvmds = MVMDS(n_components=2)
+    Xs_reduced = mvmds.fit_transform(Xs)
+
+    fig = plt.figure(figsize=figsize, **fig_kwargs)
+    sns.set_context(context)
+
+    if labels is None:
+        plt.scatter(
+            Xs_reduced[:, 0], Xs_reduced[:, 1],
+            cmap=cmap, **scatter_kwargs
+        )
+    else:
+        plt.scatter(
+            Xs_reduced[:, 0], Xs_reduced[:, 1],
+            cmap=cmap,
+            c=labels,
+            **scatter_kwargs,
+        )
+    if ax_labels:
+        plt.xlabel("Component 1")
+        plt.ylabel("Component 2")
+    if not ax_ticks:
+        plt.xticks([], [])
+        plt.yticks([], [])
+
+    if title is not None:
+        plt.tight_layout(rect=[0, 0.03, 1, 0.95])
+        plt.title(title)
```

Comment:
plt vs ax? title vs. suptitle? See above comment

Reply:
If it's only a single figure, rather than a grid, suptitle results in an off-center title. So I used plt.title here. ax.set_title resulted in some weird issues where axis labels were different sizes depending on the title